assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -65,16 +65,29 @@ const Area = styled.div`
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  padding: 16px 0 24px;
 `;
 
-const List = styled(Area)`
+const List = styled(Area).attrs({ as: 'ul', role: 'listbox' })`
```

Comment:
You could add the following properties too:

* `aria-orientation="horizontal"`, as vertical is default assumption
* `aria-label`, but remember I18N.
* `aria-activedescendant`, but that's a dynamic property
* `aria-roledescription="carousel"` as "listbox" really isn't a good description of the widget. This specifically does not require I18N as far as I know.

Reply:
From what I'm reading, `aria-roledescription` should probably be localizable as well

Reply:
As mentioned at the bottom, it will be handled in https://github.com/google/web-stories-wp/issues/10

Reply:
Make sure to add it as implementation brief notes to that issue, though.